smallrye-reactive-messaging-provider/src/test/java/io/smallrye/reactive/messaging/providers/wiring/EmitterFactoryForLiteral.java

```diff
@@ -0,0 +1,30 @@
+package io.smallrye.reactive.messaging.providers.wiring;
+
+import javax.enterprise.util.AnnotationLiteral;
+
+import org.eclipse.microprofile.reactive.messaging.Emitter;
+
+import io.smallrye.reactive.messaging.MutinyEmitter;
+import io.smallrye.reactive.messaging.annotations.EmitterFactoryFor;
+
+public final class EmitterFactoryForLiteral extends AnnotationLiteral<EmitterFactoryFor> implements EmitterFactoryFor {
```

Comment:
Nitpicking: a `Literal` class is typically added directly inside the qualifier annotation.

Reply:
👍  At first, I only needed it for tests. But I think it could be useful for Quarkus build time as well.

Reply:
it can still be added to the annotation as a static class.

Reply:
👍